keras/src/losses/losses.py

```diff
@@ -6,6 +6,7 @@
 from keras.src.losses.loss import Loss
 from keras.src.losses.loss import squeeze_or_expand_to_same_rank
 from keras.src.saving import serialization_lib
+from keras.src.utils.module_utils import optree
```

Comment:
Keras is designed to not need optree (as we found it cannot be run in some environments...). If you need a new optree function it should be added in `keras.tree` and made to work with dm-tree as well

Reply:
That's unfortunate. I make heavy use of optree's advanced features. This limitation will require some rethinking of the current implementation.

Reply:
I removed the optree dependency.